src/web/app/components/sortable-table/sortable-table.component.ts

```diff
@@ -29,12 +45,14 @@ export class SortableTableComponent implements OnInit {
   @Input()
   columns: ColumnData[] = [];
 
+  // Default to use supplied value for both sorting and displaying
+  // Use CustomTableCellData if value used for sorting is different from displaying
```

Comment:
Is the comment necessary?

> Default to use supplied value for both sorting and displaying

What is supplied value?

Reply:
Same here